frontend/src/scenes/groups/groupsListLogic.ts

```diff
@@ -50,7 +52,20 @@ export const groupsListLogic = kea<groupsListLogicType>([
     selectors({
         groupTypeName: [
             (s, p) => [s.aggregationLabel, p.groupTypeIndex],
-            (aggregationLabel, index): string => aggregationLabel(index).singular,
+            (aggregationLabel, groupTypeIndex): string =>
+                groupTypeIndex ? aggregationLabel(groupTypeIndex).singular : 'Group',
         ],
     }),
+    afterMount(({ actions, values }) => {
+        if (values.query.source.kind === NodeKind.GroupsQuery && values.query.source.select === undefined) {
+            const defaultColumns = values.groupTypes.get(values.query.source.group_type_index)?.default_columns
+            actions.setQuery({
+                ...values.query,
+                source: {
+                    ...values.query.source,
+                    select: defaultColumns ?? defaultDataTableColumns(NodeKind.GroupsQuery),
+                },
+            })
```

Comment:
This feels a bit icky. Is there a better way of setting the default columns for the query on load?